src/schemaGenerator.ts

```diff
@@ -386,14 +389,19 @@ function getFieldsForType(type: GraphQLType): GraphQLFieldMap<any, any> {
 
 function addResolveFunctionsToSchema(
   schema: GraphQLSchema,
-  resolveFunctions: IResolvers,
+  inputResolveFunctions: IResolvers,
   resolverValidationOptions: IResolverValidationOptions = {},
+  inheritResolversFromInterfaces: boolean = false
```

Comment:
Rather than adding another boolean positional argument, let's merge the validation options and the resolver inheritance option into a single options parameter. This will probably require a more general name for the `IResolverValidationOptions` type, such as `IAddResolveFunctionsToSchemaOptions`.

More on why boolean positional arguments are problematic: https://ariya.io/2011/08/hall-of-api-shame-boolean-trap